lite/examples/pose_estimation/android/app/src/main/java/org/tensorflow/lite/examples/poseestimation/ml/MoveNet.kt

```diff
@@ -51,24 +53,26 @@ class MoveNet(private val interpreter: Interpreter) : PoseDetector {
         // allow specifying model type.
         fun create(context: Context, device: Device, modelType: ModelType): MoveNet {
             val options = Interpreter.Options()
+            var gpuDelegate: GpuDelegate? = null
             options.setNumThreads(CPU_NUM_THREADS)
             when (device) {
                 Device.CPU -> {
                 }
                 Device.GPU -> {
-                    // TODO: Create a new Movenet model that can run on GPUDelegate
-//                    options.addDelegate(GpuDelegate())
+                    gpuDelegate = GpuDelegate()
+                    options.addDelegate(gpuDelegate)
                 }
                 Device.NNAPI -> options.setUseNNAPI(true)
             }
             return MoveNet(
                 Interpreter(
                     FileUtil.loadMappedFile(
                         context,
-                        if (modelType == ModelType.Lightning) "movenet_lightning_v3.tflite"
-                        else "movenet_thunder_v3.tflite"
+                        if (modelType == ModelType.Lightning) "movenet_lightning.tflite"
```

Comment:
For constants, can you make them as class constants?

Reply:
Done.